sklearn/linear_model/_ridge.py

```diff
@@ -2157,6 +2183,16 @@ def fit(self, X, y, sample_weight=None):
             Individual weights for each sample. If given a float, every sample
             will have the same weight.
 
+        **params : dict, default=None
+            Parameters to be passed to GridSearchCV or the underlying scorer.
```

Comment:
I think a better wording might be:

```suggestion
            Extra parameters for the underlying scorer.
```

since sample_weight could also be used for the scorer.